### PR TITLE
Add minor updates in ui-tests 

### PR DESCRIPTION
**Pull request by unsortedhashsets**
- Camel Debugger tests -> increase the timeout by 1 minute (based on timeout issues during tests on macOS)
- Jbang commands with automatic reload' ->  increase the timeout by 1 minute  (based on timeout issues during tests on macOS)
- Add timeout for `driver.wait` in function `waitUntilTerminalHasText` (as discussed with @mlorinc here -> https://github.com/camel-tooling/camel-dap-client-vscode/pull/407)
- Add timeout for `driver.wait` with `replaceTextInCodeEditor` (same reason^)
- Delete after/before all/each description -> tests are failed to make screenshot with after/before all/each descriptions. example: [gh_action](https://github.com/camel-tooling/camel-dap-client-vscode/actions/runs/5197356242/jobs/9372997610) , error: `Error: Artifact path is not valid: /Camel file editor test.Camel Actions.'after all' hook: After cleanup for 'Can execute 'Run Camel Application with JBang and Debug' action'.png. Contains the following character:  Colon :`
- Change extester on vscode-uitests-tooling in imports (init. extester package uninstall)
- Delete not needed `as View...`

Hope it will stabilize tests a bit more 

Flay issues:

1) Reload -> Solved
~~Flaky issue exists on MacOS -> `await editor.selectText(text)`~~
~~`Error: NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".monaco-menu"}`~~
~~Should be fixed with new release of extester (vscode-uitests-tooling)~~

2) Debugger -> Flaky but maybe because of the extension, sometimes debugger seems is not attached  correctly (https://github.com/camel-tooling/camel-dap-client-vscode/actions/runs/5211701261/jobs/9404322203)

Camel Debugger tests Update of a value with Camel debugger

```
  1) Camel Debugger tests
       Update of a value with Camel debugger:
     TimeoutError: Wait timed out after 240076ms
      at /Users/runner/work/camel-dap-client-vscode/camel-dap-client-vscode/node_modules/selenium-webdriver/lib/webdriver.js:929:17
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

3) Install test -> Sometimes happens (https://github.com/camel-tooling/camel-dap-client-vscode/actions/runs/5211701261/jobs/9404620692)

Install test Marketplace presentation Has correct title

```
  1) Install test
       Marketplace presentation
         Has correct title:
     StaleElementReferenceError: stale element reference: element is not attached to the page document
  (Session info: chrome=102.0.5005.196)
      at Object.throwDecodedError (node_modules/selenium-webdriver/lib/error.js:524:15)
      at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:601:13)
      at Executor.execute (node_modules/selenium-webdriver/lib/http.js:529:28)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Driver.execute (node_modules/selenium-webdriver/lib/webdriver.js:745:17)
```

### CODE REVIEW

**Review by mlorinc — CHANGES_REQUESTED** [review]

File: src/ui-test/utils.ts [hunk 1]
```diff
@@ -121,6 +121,7 @@ export async function disconnectDebugger(driver: WebDriver, interval = 500): Pro
             await driver.wait(until.elementIsNotVisible(debugBar));
```
[the remark this answers is not in the document]
Comment (unsortedhashsets) [reply src/ui-test/utils.ts]: Done, thanks!

**Review by djelinek — CHANGES_REQUESTED** [review]

File: src/ui-test/tests/context.menu.test.ts [hunk 2]
```diff
@@ -6,11 +6,9 @@ import {
     before,
     EditorView,
     SideBarView,
-    ViewControl,
-    ViewSection,
     VSBrowser,
     WebDriver,
-} from 'vscode-extension-tester';
+} from 'vscode-uitests-tooling';
```
[the remark this answers is not in the document]
Comment (unsortedhashsets) [reply src/ui-test/tests/context.menu.test.ts:11]: Done

**Comment by unsortedhashsets** [comment]
@djelinek @mlorinc 
Hi, guys, please look at the description of the current flaky tests in the description of the PR
Seems the `debugger` is flaky due to the extension issue, I was able once to reproduce it locally, but it is more unstable on macos/windows

**Comment by djelinek** [comment]
@unsortedhashsets nice investigation! thx

ad 2) and 3) feel free to open issues in JIRA for that and we can investigate and stabilize later in next iteration in case that is not happening very often

**Comment by djelinek** [comment]
@unsortedhashsets rebase required

**Comment by djelinek** [comment]
@unsortedhashsets could you please open new bug on extester project for this `Delete after/before all/each description -> tests are failed to make screenshot with after/before all/each descriptions. `.. it should work in my opinion

**Review by djelinek — APPROVED** [review]

**Review by mlorinc — APPROVED** [review]

**Pull request merged**